### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by ghost**
Correct all error in the dev, and change style of element

### CODE REVIEW

**Review by Elfaram — DISMISSED** [review]

File: frontend/src/components/CardForm/index.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          onClick={() => {
+            if (isActive === false) {
+              addStorage();
+            } else {
+              deleteStorage();
+            }
+            handleIsActive();
+          }}
+        >
+          <img className="favorite" src={isActive ? love : liket} alt="" />
```
Comment (LuigiMenez) [inline frontend/src/components/CardForm/index.jsx]: Missing the alt

File: frontend/src/components/CarrousselNews/index.jsx [hunk 2]
```diff
@@ -31,7 +31,7 @@ function CarrousselNews() {
           {moviesData.map((movie) => (
             <img
               key={movie.id}
-              src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
+              src={`https://image.tmdb.org/t/p/original${movie.backdrop_path}`}
               alt=""
```
Comment (LuigiMenez) [inline frontend/src/components/CarrousselNews/index.jsx]: Missing the alt

File: searBarv4 [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  hotfix[m
+  menu_burger[m
+  result-page[m
+  searchBarv4[m
+  searchbar[m
+  searchbarV2[m
+  searchbarV3[m
+  shearchbar[m
+  test-dev[m
+  test-searchbar[m
```
Comment (LuigiMenez) [inline searBarv4]: WHAT??????

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Missing the alt, and Has the RP been proofread?

**Review by Elfaram — APPROVED** [review]

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/CardForm/index.jsx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    <SCardForm>
+      <div className="card">
+        <Link to={`/detail/${movie.id}`}>
+          <img
+            src={
+              movie.poster_path
+                ? `https://image.tmdb.org/t/p/w500${movie.poster_path}`
+                : "./img/poster.jpg"
+            }
+            alt="affiche film"
```
Comment (LoicBrassart) [inline frontend/src/components/CardForm/index.jsx:44]: (Your site should be in English)

File: frontend/src/components/CardForm/style.jsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    border: none;
+  }
+  .favorite {
+    width: 2rem;
+    color: aliceblue;
+  }
+  #emptyHearth {
+    color: red;
+  }
+  #fullHearth {
```
Comment (LoicBrassart) [inline frontend/src/components/CardForm/style.jsx:32]: Should not use ID (several cards -> several identical IDs on your page)

File: frontend/src/components/CarrousselNews/index.jsx [hunk 6]
```diff
@@ -31,8 +31,8 @@ function CarrousselNews() {
           {moviesData.map((movie) => (
             <img
               key={movie.id}
-              src={`https://image.tmdb.org/t/p/w500${movie.backdrop_path}`}
-              alt=""
+              src={`https://image.tmdb.org/t/p/original${movie.backdrop_path}`}
+              alt="poster film"
```
Comment (LoicBrassart) [inline frontend/src/components/CarrousselNews/index.jsx:35]: (English plz !)

File: frontend/src/components/Form/style.jsx [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+      margin: 8px auto;
+      width: 220px;
+      display: flex;
+      justify-content: space-around;
+      cursor: pointer;
+
+      #goodToBad {
+        border-radius: 10px 0 0 10px;
+      }
+      #badToGood {
```
Comment (LoicBrassart) [inline frontend/src/components/Form/style.jsx:56]: No IDs

File: frontend/src/components/Form/style.jsx [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+      input[type="submit"] {
+        padding: 1px 10px;
+        width: 220px;
+        border-radius: 0 0 10px 10px;
+        border: none;
+        background: $color-1;
+        color: $white;
+        font-size: 1.5rem;
+        cursor: pointer;
+        z-index: 10;
```
Comment (LoicBrassart) [inline frontend/src/components/Form/style.jsx:38]: You know better than that

File: frontend/src/components/Form/style.jsx [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+      cursor: pointer;
+
+      #goodToBad {
+        border-radius: 10px 0 0 10px;
+      }
+      #badToGood {
+        border-radius: 0 10px 10px 0;
+      }
+      #goodToBad,
+      #badToGood {
```
Comment (LoicBrassart) [inline frontend/src/components/Form/style.jsx:60]: Same

File: frontend/src/components/Form/style.jsx [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+      #goodToBad span {
+        transform: translateY(-50%) rotate(-90deg);
+        right: 18px;
+      }
+      #badToGood span {
+        transform: translateY(-50%) rotate(90deg);
+        left: 16px;
+      }
+      #goodToBad span,
+      #badToGood span {
```
Comment (LoicBrassart) [inline frontend/src/components/Form/style.jsx:79]: Same

File: frontend/src/components/Header/style.jsx [hunk 11]
```diff
@@ -130,6 +130,36 @@ export default styled.header`
...
+    font-size: 17px;
+    border: none;
+    cursor: pointer;
+  }
+
+  .topnav .search-container button:hover {
+    background: #ccc;
+  }
+
+  #searchLogo {
```
Comment (LoicBrassart) [inline frontend/src/components/Header/style.jsx:160]: Same

**Review by LoicBrassart — APPROVED** [review]
Beware of this kind of PR: should I be stricter, I'd refuse it for trivial things and all your fixes would be wasted :(

**Pull request merged**